src/stp/new_tactics/Formation.cpp

```diff
@@ -28,7 +28,12 @@ std::optional<StpInfo> Formation::calculateInfoForSkill(StpInfo const &info) noe
     StpInfo skillStpInfo = info;
 
     // Be 100% sure the angle is 0 during the formation
-    skillStpInfo.setAngle(0.0001);
+    if (info.getAngle() == Angle()) {
```

Comment:
We need an .isDefault() method on the geometric classes.

Reply:
I indeed agree with Johan